test/browser/test.js

```diff
@@ -146,6 +146,13 @@ function runTest(name, autoTest) {
 
```

Comment:
Hi @pragmadash, you can add your own separate tests by creating another `it()` block in `morphdom/test/browser/test.js`. For example, here is a test for `onBeforeMorphEl()` that you can add as a guide: https://github.com/patrick-steele-idem/morphdom/blob/be3e5793c94175434b2aa256c6e3d1daeb98d9a9/test/browser/test.js#L363-L370 

Thanks!